server/src/tenant/metadata/field-metadata/field-metadata.util.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Generate a column name from a field name removing unsupported characters.
```

Comment:
Probably won't keep this format as described in my previous PR since we prefer to have unspecified names for columns such as "field_1", "field_2", ...